jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -222,6 +222,17 @@ def on_limits_change(self, *args):
             return
         self.set_compass(self.state.layers[i].layer)
 
+    @property
+    def top_visible_data_label(self):
```

Comment:
@kecnry like this? I also put this in the `center_on` docstring below.

Reply:
Perfect, thanks!